edelphi-persistence/src/main/java/fi/metatavu/edelphi/dao/panels/PanelDAO.java

```diff
@@ -116,6 +123,41 @@ public List<Panel> listByDelfoiAndUser(Delfoi delfoi, User user) {
     
     return entityManager.createQuery(criteria).getResultList();
   }
+
+  public List<Panel> listPanelsToArchive(Date before, int maxResults) {
```

Comment:
Method names in DAOs should reflect what they do, not why the exist, so rename this to listPanelsByStateArchivedAndLastModifiedBefore